ios/engine/KMEI/KeymanEngine.xcodeproj/xcshareddata/xcschemes/KME-iphoneos.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "0830"
+   LastUpgradeVersion = "0900"
```

Comment:
Question - do we plan to keep these schemes around?  If we're instead relying upon a framework, the device-specific targets should at least be deprecated, no?

Reply:
These don't work anymore so removing them.